src/main/resources/db/migration/V1_1__createTables.sql

```diff
@@ -25,7 +25,15 @@
         primary key (family_id)
     ) engine=InnoDB;
 
+    create table shedlock(
```

Comment:
I do not know the history of this service, but shouldn't the add of this table be in an own migration file (V1_2)? Or will you wipe the test-db and re-create all tables again?

Reply:
I'll will wipe the test-db and recreate it. The service is never deployed to prod.